pandas/tests/computation/test_eval.py

```diff
@@ -314,7 +314,10 @@ def check_alignment(self, result, nlhs, ghs, op):
 
             # direct numpy comparison
             expected = self.ne.evaluate(f"nlhs {op} ghs")
-            tm.assert_numpy_array_equal(result.values, expected)
+            if np.issubdtype(result.values.dtype, np.floating):
+                tm.assert_almost_equal(result.values, expected)
+            else:
+                tm.assert_numpy_array_equal(result.values, expected)
```

Comment:
this is for the Travis failure thats currently failing every PR right, i.e. not related to this PR?  if so, pls revert

Reply:
@jbrockmendel yes